src/components/MentionTextArea/index.tsx

```diff
@@ -147,6 +147,7 @@ const MentionTextArea = ({
     [initialContent, disabled]
   );
 
+  console.info(editor?.getHTML());
```

Comment:
Can probs nix this or at least add a `TODO` to remove it before it hits `main`

Reply:
thanks for finding and fixing this